tools/packaging/common/envoy_bootstrap.json

```diff
@@ -78,7 +78,7 @@
     "stats_tags": [
       {
         "tag_name": "cluster_name",
-        "regex": "^cluster\\.((.+?(\\..+?\\.svc\\.cluster\\.local)?)\\.)"
+        "regex": "cluster(\\.(.+);)"
```

Comment:
Wouldn't this break with older data plane? If a new control plane sends ";" then it's going break on older Envoys.

Reply:
Don't we require a restart when upgrading for that reason? On restart, the new bootstrap will be sent to the new pod. In place control plane upgrade will break through yes

Reply:
Istio traditionally supported version N control plane with N-1 data plane. This violates that contract.

Reply:
Hmm this is true...a feature flag doesn't make things much better. I can play around with the regex to attempt to make it still match the old style